resource/decker/support/plugins/feedback/feedback.js

```diff
@@ -466,32 +464,42 @@ class Feedback {
   </div>
   <div class="feedback-controls">
     <div class="feedback-controls-wrapper">
-      <span class="votes" title="${text.votes}" aria-label="${text.votes}">${comment.votes > 0 ? comment.votes : ""
-      }</span>
-      <button class="${comment.didvote ? "fas" : "far"} fa-thumbs-up vote ${!isAuthor ? "canvote" : "cantvote"
-      } ${comment.didvote ? "didvote" : ""}"
+      <span class="votes" title="${text.votes}" aria-label="${text.votes}">${

```

Comment:
Why are there so many changes? Is this pure formatting? Formatting changes caused merge conflicts earlier, so let's avoid this by using the same code formater (Prettier)